### ISSUE TITLE
Drawing tool :  show length for each segment of a polygon perimeter

### ISSUE THREAD

**Issue by maltaesousa** [camptocamp/GeoMapFish#52]
### Who requested this new feature?
> SITN

### Is your feature request related to a problem?
> No

### Describe the solution you'd like
> While drawing a polygon, length of currently drawn segment is shown.
> After drawing a polygon, a checkbox "Show length" will show the lengths of each segment of a polygon

### Additional information or points to watch out for

![image](https://user-images.githubusercontent.com/3328875/125073889-e7a8b980-e0bc-11eb-8b80-08fadf3ba9eb.png)

![image](https://user-images.githubusercontent.com/3328875/125073504-5b969200-e0bc-11eb-93b6-3ab153956402.png)

**Comment by remyguillaume** [comment]
I agree, this is a greate improvement (if we decide to migrate the drawing component)
